diff --git a/node_modules/cosmiconfig/dist/loaders.js b/node_modules/cosmiconfig/dist/loaders.js
index b71d7fc..5643e8f 100644
--- a/node_modules/cosmiconfig/dist/loaders.js
+++ b/node_modules/cosmiconfig/dist/loaders.js
@@ -19,8 +19,7 @@ const loadJsSync = function loadJsSync(filepath) {
 exports.loadJsSync = loadJsSync;
 const loadJs = async function loadJs(filepath) {
     try {
-        const { href } = (0, url_1.pathToFileURL)(filepath);
-        return (await import(href)).default;
+        return (await import(`${filepath}?t=${Date.now()}`)).default;
     }
     catch (error) {
         try {
